ftw/subsite/viewlets/analytics.py

```diff
@@ -0,0 +1,31 @@
+from Acquisition import aq_inner, aq_parent
+from Products.CMFPlone.interfaces import IPloneSiteRoot
+from ftw.subsite.interfaces import ISubsite
+from plone import api
+from plone.app.layout.analytics.view import AnalyticsViewlet
+
+
+class SubsiteAnalyticsViewlet(AnalyticsViewlet):
+    def update(self):
+        self.csutom_analytics_snippet = self.get_custom_analytics_snippet()
+
+    def render(self):
+        default_snippet = super(SubsiteAnalyticsViewlet, self).render()
+        return self.csutom_analytics_snippet or default_snippet
+
+    def get_custom_analytics_snippet(self):
+        obj = self.context
+        while not IPloneSiteRoot.providedBy(obj):
+
+            if ISubsite.providedBy(obj) and obj.Schema()['webstatsAnalyticsSnippet'].get(obj):
+                snippet = obj.getRawWebstatsAnalyticsSnippet()
```

Comment:
As you have removed the safe-html cleanup (previous discussion) you should now be able to use the regular, non-raw getter and still be allowed to use the script tag.
I'd prefer not using the raw getter since this is a more internal thing :wink: 
Can you try that?